code/game/objects/_items.dm

```diff
@@ -1005,4 +1007,72 @@ THIS SCOPE CODE IS DEPRECATED, USE AIM MODES INSTEAD.
 
 
 /obj/item/repair_needed()
-	return max_health - health
\ No newline at end of file
+	return max_health - health
+
+/obj/item/proc/set_force_string()
```

Comment:
this definitely doesn't need to be cached, just run the proc to fetch it on demand

and don't hardcode numbers here, use the same defines that weapon forces are derived from

Reply:
> this definitely doesn't need to be cached, just run the proc to fetch it on demand
> 
> and don't hardcode numbers here, use the same defines that weapon forces are derived from

Reverted